careers/base/templates/base.jinja

```diff
@@ -2,13 +2,23 @@
 <html lang="{{ LANG }}" dir="{{ DIR }}" class="no-js"
       data-ga-code="{{ settings.GA_ACCOUNT_CODE }}"
       data-gtm-code="{{ settings.GTM_ACCOUNT_CODE }}">
+{% set default_slug = "Mozilla Careers &mdash; Feel good about your work again" %}
```

Comment:
I'd move this to the very first line to make it easier to spot

Reply:
Fixed